solution/operator/role_binding.yaml

```diff
@@ -0,0 +1,11 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: zenko-operator
+subjects:
+- kind: ServiceAccount
+  name: zenko-operator
+roleRef:
+  kind: Role
+  name: zenko-operator
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
We don't need this, MetalK8s will take care of `RoleBinding`.

Reply:
Removed the file